engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/registrar/FlinkProcessRegistrar.scala

```diff
@@ -65,12 +61,17 @@ class FlinkProcessRegistrar(compileProcess: (EspProcess, ProcessVersion, Deploym
 
   protected def isRemoteEnv(env: StreamExecutionEnvironment): Boolean = env.getJavaEnv.isInstanceOf[RemoteStreamEnvironment]
 
-  protected def usingRightClassloader(env: StreamExecutionEnvironment)(action: => Unit): Unit = {
+  //In remote env we assume FlinkProcessRegistrar is loaded via userClassloader. If this is not the case,
```

Comment:
I don't exactly understand the part with "if this is no the case, (...) should be overloaded". You have some concrete examples when it is not the case?

Reply:
I removed this part of comment, it was kind of misleading...